lib/irb/pager.rb

```diff
@@ -30,6 +40,21 @@ def page
 
       private
 
+      def content_exceeds_screen_height?(content)
+        screen_height, screen_width = begin
+          Reline.get_screen_size
+        rescue Errno::EINVAL
```

Comment:
Maybe we should also fix Reline to rescue EINVAL? Reline only rescue ENOTTY.

I found a comment for EINVAL reproducible environment here
https://github.com/ruby/ruby/blob/26aef1c73639718bc00c271ce7176bf3ee565c12/test/io/console/test_io_console.rb#L310
and here
https://bugs.ruby-lang.org/issues/10494